SPECK/src/main/java/SPEck.java

```diff
@@ -25,28 +25,43 @@
  */
 
 public class SPEck implements Serializable {
+
+    /**
+     * Initializer
+     * @param   scc    <JavaSparkContext> spark context for parallel execution
+     * @param   fileIn  <String> storing the file name of the input dataset to mine the SFSP
```

Comment:
Let's use "observed dataset", instead of "input dataset", and let's rename the variable to `obsDatasetPath`, and refer to "path", rather than "file name"